info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -283,6 +288,10 @@ private void storeElectionContestCandidateInDatabase(JsonObject candidate,
         Entity.newBuilder(candidateKey)
             .set("name", name)
             .set("partyAffiliation", party + " Party")
+            .set("email", email);
+            .set("phone number",phoneNumber);
```

Comment:
Are you referring to the first parameter not matching the second on line 292? This too has a lot to do with the database having different column name formatting. I can alter the database to utilize lower camel case but it will break existing code. It should be an easy fix however.

Reply:
I meant that in our database, all other column names are named using camel case. It might be nice to keep a consistent style